composer.json

```diff
@@ -12,15 +12,15 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
-        "phpunit/phpunit": "^4.0|^5.0"
+        "php": "^7.0",
```

Comment:
uhm ok so should this support both phpunit 5 and phpunit 6? I thought the bump would've caused a new major to be released, but I will check if it's doable to support both at the same time :)